luatest/server_v2.lua

```diff
@@ -492,7 +492,7 @@ end
 --- Wait for the server to reach at least the same vclock as the other server.
 -- Not including the local component, of course.
 --
--- @table other_server Other server's object.
+-- @param other_server (table) Other server's object.
```

Comment:
Resolves the followring error

>  id tag: 'name" has trailing text ; use not_luadoc=true if you want description to continue between tags

https://github.com/lunarmodules/LDoc/issues/66

Reply:
Fixed.